tests/FakeItEasy.Tests/FakeFacadeTests.cs

```diff
@@ -89,6 +89,18 @@ public void ClearConfiguration_should_be_guarded()
         }
 
         [Fact]
+        public void ClearRecordedCalls_should_be_guarded()
+        {
+            // Arrange
+
+            // Act
+
+            // Assert
+            Expression<System.Action> call = () => this.facade.ClearRecordedCalls(A.Dummy<object>());

```

Comment:
Shouldn't this statement be in the Arrange phase?

In fact, the `.Should().BeNullGuarded()` method doesn't lend itself very well to AAA style tests, since it does the Act and Assert phases in one instruction...